src/app/_components/Popup.tsx

```diff
@@ -0,0 +1,76 @@
+'use client';
+
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import { Autoplay, Pagination } from 'swiper/modules';
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+const Popup = ({
+  setSee,
+}: {
+  setSee: Dispatch<SetStateAction<string | null>>;
+}) => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  useEffect(() => {
+    setSee(localStorage.getItem('see'));
+  }, [setSee]);
+
+  const handleSeeAgain = () => {
+    if (isChecked) {
+      localStorage.setItem('seenPopup', 'true');
+    }
+    setSee('true');
+  };
+
+  const handleCheckbox = () => {
+    isChecked ? setIsChecked(false) : setIsChecked(true);
+  };
+
+  return (
+    <div className='absolute bottom-0 z-50 flex w-full flex-col bg-dark-accent2'>
```

Comment:
```suggestion
    (see && <div className='absolute bottom-0 z-50 flex w-full flex-col bg-dark-accent2'> ) 
```

와 같이 return문에 연산자를 추가해주고 see를 props가 아닌  Popup 컴포넌트 내에서 state를 관리해준다면 page단위에서 see와 setSee가 필요없을 것으로 예상됩니다. (자연스럽게 `use client`도 지울 수 있을 것으로 예상되구요)

혹시 제가 작성해놓은게 이해가 잘 안되신다면 카톡으로 호출해주세요 ! 화면공유해서 제가 생각한 코드를 보여드리는게 빠를 것 같아서요 ! 논의가 계속 길어지는데 감사합니다.

Reply:
그렇게 하면 에러납니다 그래서 하지못했던 것이지요

Reply:
음 그럼 일단 merge하고 나중에 재작업 해보겠습니다 !